portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java

```diff
@@ -6248,6 +6252,8 @@ protected User doCheckLockout(User user, PasswordPolicy passwordPolicy)
 		}
 
 		if (user.isLockout()) {
+			checkLoginFailure(user);
+
```

Comment:
Calling this here will cause every request to render personal_information.jsp to increment the `failedLoginAttempts` for a locked out account. It may not cause any direct problems, but it seems wrong.

Reply:
I added a solution that does not call the method, but the user = userPersistence.update(user) will be called later with other possible updates.